.github/workflows/pr.yml

```diff
@@ -205,7 +205,7 @@ jobs:
         run: yarn install --frozen-lockfile
 
       - name: Check for change files
-        run: node ./node_modules/beachball/bin/beachball.js check --changehint "Run `yarn change` from root of repo to generate a change file."
+        run: node ./node_modules/beachball/bin/beachball.js check --branch origin/$(Build.SourceBranchName) --changehint "Run `yarn change` from root of repo to generate a change file."
```

Comment:
Build.SourceBranchName is 'merge' on PRs.  So I dont think this is doing the right thing..

Reply:
There is System.PullRequest.TargetBranch which is similar but will be 'refs/heads/master' rather than just 'master'

Reply:
Looks like you're right. I'll try to find if there's an env variable we can still use (Haven't tried to make things super correct for the GH version though, but we see this with the ADO one).

Reply:
Turns out the docs lie, and the output of System.PullRequest.TargetBranch is just master. Makes things easier for us I guess.

Reply:
Filed a bug at https://github.com/microsoft/azure-pipelines-tasks/issues/12588 on this being inconsistent with documentation. For now this seems to work though.